onnxruntime/core/providers/tensorrt/tensorrt_execution_provider_helper.cc

```diff
@@ -258,4 +258,29 @@ void TensorrtExecutionProvider::SetAllGraphInputs(Graph& graph) const {
 
   graph.SetInputs(graph_inputs_including_initializers);
 }
+
+// Check if DDS op is in the ComputeCapability/subgraph.
+void TensorrtExecutionProvider::CheckDDSOpInSubGraph(const GraphViewer& graph,
+                                                     std::vector<std::unique_ptr<ComputeCapability>>& compute_capabilities,
+                                                     const std::unordered_set<std::string>& dds_op_set,
+                                                     int32_t trt_version) const {
+  auto is_dds_op = [&](const auto& node) {
+    if (dds_op_set.find(node->OpType()) != dds_op_set.end()) return true;
+    return false;
+  };
+
+  for (auto& compute_capability : compute_capabilities) {
+    auto& indexed_sub_graph = compute_capability->SubGraph();
+    for (auto i : indexed_sub_graph->Nodes()) {
+      if (is_dds_op(graph.GetNode(i))) {
+        if (trt_version >= 100000 && trt_version < 110000) {
```

Comment:
dds is excluded by TRT built in parser before TRT 10.6 right? 
what versions do we need to output this check for?

Reply:
User still can use OSS parser branch (<= TRT 10.6) that has DDS ops enabled.
I think until Nvidia provides the explanation regarding the performance improvement we saw (> TRT 10.5), it's better to have this message.

Reply:
I don't think we want to optimize too much for the combination of OSS parser and older TRT versions. 
if someone is going to build from source, it's more likely they are doing it to support the newest code/updates. 
my concern is the message will get thrown even if someone is using built-in parser with older TRT version (which will be the more common case)

Reply:
Re: my concern is the message will get thrown even if someone is using built-in parser with older TRT version (which will be the more common case)
Hmm, i assume you mean built-in parser with newer TRT version?

Right now the logic checks whether the node is DDS and TRT version is 10. 
Do you think it's better to limit the check to TRT version > 0 and <= 10.5? Or we don't output this message at all?

Reply:
debating how useful this message is. it's verbose so it has to be explicitly enabled. 
and I wonder if it may cause some confusion because we output regardless of whether there really is a perf impact or not.

Reply:
Removed the message and added in the doc instead.